code/game/machinery/soulbencher.dm

```diff
@@ -0,0 +1,45 @@
+/obj/machinery/soulbencher
+	name = "soulbencher"
+	desc = "I gave up trying to go back a long time ago. \
```

Comment:
In all seriousness the only point of having this be a machine in my PR was so I could grab them as soon as they became a ghost. You don't need a machine in this PR since you are fine with them becoming a ghost. This could all be done with a var on their mind.